src/WorkingStrings.java

```diff
@@ -0,0 +1,18 @@
+public class WorkingStrings {
+
+    public static void main(String[] args) {
+
+        String s = "Hello World!";
+
+        System.out.println("Länge: " + s.length());
+        System.out.println("In Gross: " + s.toUpperCase());
+        System.out.println("In klein: " + s.toLowerCase());
+        String newWord = "Coders Bay";
+        s = s.substring(0, 5);
+        for (int i=0; i<15; i++) {
+            System.out.println(newWord + " " + s);
+        }
+    }
+
```

Comment:
anstelle einer Schleife wollte ich hier darauf hinaus, dass ihr die `repeat` Methode auf dem String ausprobiert

Reply:
Hi Pia.

Habe es leider nicht geschafft dass ich ohne Schleife einen Zeilenumbruch hinbekomme. Habe die repeat-Methode angewendet, aber die listet alles in einer Zeile..